Makefile

```diff
@@ -1,125 +1,168 @@
-# Project Environment Variables
+############################################################
+# Makefile for the Anchore CLI, a simple command line interface to the
+# Anchore Engine service. The rules, directives, and variables in this
+# Makefile enable testing, Docker image generation, and pushing Docker
+# images.
+############################################################
+
+
+#### Docker Hub, git repos
+############################################################
 DEV_IMAGE_REPO := anchore/anchore-cli-dev
+PROD_IMAGE_REPO := anchore/anchore-cli
+# TEST_HARNESS_REPO := https://github.com/anchore/test-infra.git
+TEST_HARNESS_REPO := https://github.com/robertp/test-infra.git
+
 
-# Environment variables set in CircleCI environment
+#### CircleCI environment variables
+############################################################
 export VERBOSE ?= false
 export CI ?= false
-# DOCKER_USER & DOCKER_PASS are declared in CircleCI contexts
```

Comment:
Can you leave these comments? They're stored in 2 different places. DOCKER_USER/PASS is in a context, which is shared across projects. And the LATEST_RELEASE_BRANCH is in the project env vars, which is specific to each project.

Reply:
No problem, comments back in place